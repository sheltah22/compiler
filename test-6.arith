if true
  then (2 * 3)
  else  2 * 1 - 1
false
