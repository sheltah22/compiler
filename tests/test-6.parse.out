(let x : int = 5 in (* 2 x))
